src/radical/pilot/bootstrapper/default_bootstrapper.sh

```diff
@@ -1414,15 +1414,31 @@ profile_event 'cleanup done'
 echo "#"
 echo "# -------------------------------------------------------------------"
 
-echo
-echo "# -------------------------------------------------------------------"
-echo "#"
-echo "# Tarring profiles ..."
-PROFILES_TARBALL="profiles.tgz"
-tar -czf $PROFILES_TARBALL *.prof
-ls -l $PROFILES_TARBALL
-echo "#"
-echo "# -------------------------------------------------------------------"
+if ! test -z "`ls *.prof 2>/dev/null`"
+then
+    echo
+    echo "# -------------------------------------------------------------------"
+    echo "#"
+    echo "# Tarring profiles ..."
+    PROFILES_TARBALL="$PILOTID.prof.tgz"
```

Comment:
because that makes out-of-band download simpler.  That is probably more important for the log tarball, which we expect people to provide for tickets, but then I like symmetry as you know, so did that for both tarballs, as the prof one needed renaming on the other end anyway...